terraform/modules/oidc/_data.tf

```diff
@@ -28,6 +28,7 @@ data "aws_iam_policy_document" "github_assume_role" {
 
 # tfstate and storage policy
 # trivy:ignore:AVD-AWS-0057
+# trivy:ignore:AVD-AWS-0345
```

Comment:
Does this policy need full S3 access? I'd like to move toward addressing potential issues rather than ignoring them for later use. If this needs full S3, can we add a comment somewhere in the code for that?

Reply:
This new `ignore` is due to a new Trivy policy. I've created [this ticket](https://github.com/CDCgov/dibbs-aws/issues/67) to push for improvement and better restrictions on our OIDC policies.